V2/src/data_logging.cpp

```diff
@@ -30,6 +109,7 @@ void data_logging_thread() {
         open_log.puts(buffer);
         ThisThread::sleep_for(0.1);
 
+        ThisThread::sleep_for(3s);
```

Comment:
does the "s" actually work here?

Reply:
The internet told me it would, but if it doesn't I'll just change it when I'm testing